code/datums/weather/weather_types.dm

```diff
@@ -184,3 +184,33 @@
 		status_signal.data["picture_state"] = "radiation"
 
 	frequency.post_signal(src, status_signal)
+
+
+/datum/weather/acid_rain
+	name = "acid rain"
+	desc = "Some stay dry and others feel the pain"
+
+	telegraph_duration = 400
+	telegraph_message = "<span class='danger'>Stinging droplets start to fall upon you..</span>"
+	telegraph_sound = 'sound/lavaland/ash_storm_windup.ogg'
```

Comment:
@xxalpha https://www.dropbox.com/sh/9cjmpp8n7jmqngl/AAAGOpZOEme2OfQ6coZdDidqa?dl=0

Here's a bunch of Royalty Free rain sounds i've had in my collection for a while. Any of these work? Need me to edit any of them?